yt_dlp/extractor/dropout.py

```diff
@@ -0,0 +1,101 @@
+# coding: utf-8
+from .common import InfoExtractor
+from .vimeo import VHXEmbedIE
+from ..utils import urlencode_postdata
+
+
+class DropoutIE(InfoExtractor):
+    _LOGIN_URL = 'https://www.dropout.tv/login'
+    _LOGOUT_URL = 'https://www.dropout.tv/logout'
+    _NETRC_MACHINE = 'dropout'
+
+    _VALID_URL = r'https?://(?:www\.)?dropout\.tv/videos/(?P<id>.+)'
+    _TESTS = [{
+        'url': 'https://www.dropout.tv/videos/misfits-magic-holiday-special',
+        'md5': 'c30fa18999c5880d156339f13c953a26',
+        'info_dict': {
+            'id': '1915774',
+            'display_id': 'misfits-magic-holiday-special',
+            'ext': 'mp4',
+            'title': 'Misfits & Magic Holiday Special',
+            'description': 'The magical misfits spend Christmas break at Gowpenny, with an unwelcome visitor.',
+            'release_date': '20211215',
+            'thumbnail': 'https://vhx.imgix.net/chuncensoredstaging/assets/d91ea8a6-b250-42ed-907e-b30fb1c65176-8e24b8e5.jpg',
+            'duration': 11698,
+            'uploader_id': 'user80538407',
+            'uploader_url': 'https://vimeo.com/user80538407',
+            'uploader': 'OTT Videos'
+        },
+        'expected_warnings': ['Ignoring subtitle tracks found in the HLS manifest']
+    }]
+
+    def _get_authenticity_token(self, id: str):
+        signin_page = self._download_webpage(self._LOGIN_URL, video_id=id,
+                                             note='Getting authenticity token')
+        authenticity_token = self._html_search_regex(
+            r'name=["\']authenticity_token["\'] value=["\'](.+?)["\']',
+            signin_page, 'authenticity_token')
+        return authenticity_token
+
+    def _login(self, id: str):
```

Comment:
remove `id` from this function.
You are calling it once anyways that too with `None` hardcoded.
```suggestion
    def _login(self):
```